_includes/navbar.html

```diff
@@ -25,10 +25,10 @@
                             </a>
                         {% endif %}
 
-                        <ul class="dropdown-menu{% if forloop.last %} dropdown-menu-right{% endif %}" aria-labelledby="drop{{item.index}}">
+                        <ul class="dropdown-menu sub-navbar{% if forloop.last %} dropdown-menu-right{% endif %}" aria-labelledby="drop{{item.index}}">
                             {% for entry in item.subfolderitems %}
                                 <li>
-                                    <a class="nav-link" href="/{{ entry.url }}">{{ entry.page }}</a>
+                                    <a class="nav-sublink" href="/{{ entry.url }}">{{ entry.page }}</a>
```

Comment:
'nav-link' class should not be removed, since the class is still referenced in _menu.scss.

Reply:
Since we're using bootstrap, we should also try to stick to their class names (see for example their Navbar component documentation: https://getbootstrap.com/docs/4.0/components/navbar)

Reply:
Hm not sure what design draft you're referring to. :) If the design need to change, it should be noted in the ticket. There are currently many incomplete design drafts in Figma, so those shouldn't be referenced unless the designs are attached to the GitHub ticket.

Reply:
@PaulineNemchak the important point here is to follow the design of proposed and the standards defined in the website. 

Rename the class does not solve the issue also does not improve readability. But does change the "contract" or the name patterns we are following from bootstrap.

Reply:
@PaulineNemchak I just checked the [Bootstrap nav doocumentation](https://getbootstrap.com/docs/4.0/components/navbar/#nav) again, and this 'nav-link' class should be corrected to 'dropdown-item'. That class will 'automatically' add the bootstrap hover style.

Reply:
Hi @PaulineNemchak As discussed earlier with @dricazenck , the suitable approach is to follow [Bootstrap's dropdown menu documentation](https://getbootstrap.com/docs/4.0/components/navbar/#nav). I.e. the dropdown nav link class needs to be fixed to use the 'dropdown-item' (see attached Bootstrap screenshot). It's common to use different classes for parent and child links in this case. Top-level parent links also have different hover styles, so it's appropriate to have a separate 'dropdown-item' class for the dropdown child links.
<img width="1069" alt="Screenshot 2024-05-06 at 17 35 39" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/26842896/2e4ca3c2-6c13-4e16-98c3-c8756e55e983">

Reply:
@PaulineNemchak perfect, thank you!